crates/binjs_io/src/entropy/rw.rs

```diff
@@ -161,8 +161,9 @@ pub type NameData = SmallVec<[u8; NAME_MAX_LEN]>;
 ///
 /// `TableRef` values are represented as follows:
 /// - a value `0` represents an `TableRef::Prelude` with index 1 + the latest `TableRef::Prelude` value we have encoded/decoded;
-/// - a value `v` in `[1, length of the shared dictionary]` represent `TableRef::Shared` with index `v - 1`;
-/// - a value `v > length of the shared dictionary` represents `TableRef::Prelude` with index `v - 1`.
+/// - a value `v` in `[1, window len]` represents a repeat of a previous value: 1 it the latest value, 2 the second latest, etc.
+/// - a value `v` in `[window len + 1, window len + length of the shared dictionary]` represent `TableRef::Shared` with index `v - window len - 1`;
+/// - a value `v > window len + length of the shared dictionary` represents `TableRef::Prelude` with index `v - window len - 1`.
```

Comment:
why do we use `v - window len - 1` instead of `v - window len - length of the shared dictionary - 1` ?
can you explain it somewhere?  (otherwise this now looks wrong copy-paste)

Reply:
Wrong copy-paste, you're right.